web/source/osk/browser/pendingMultiTap.ts

```diff
@@ -111,6 +111,8 @@ namespace com.keyman.osk {
       //       description.
       let e = text.KeyEvent.constructNullKeyEvent(this.vkbd.device);
       e.kNextLayer = 'caps';
+      e.Lstates = text.Codes.stateBitmasks.CAPS;
+      e.LmodifierChange = true;
```

Comment:
These two values were required to allow KeymanWeb to track the shift to the Caps Lock state. My feeling is that they *shouldn't* be required, but there's still a bit of legacy fluff around modifier / lock state that is complicating things. Easiest to add them in for now.